dist/es5src/drivers/indexeddb.js

```diff
@@ -0,0 +1,576 @@
+(function (global, factory) {
+    if (typeof define === 'function' && define.amd) {
+        define('asyncStorage', ['exports', 'module'], factory);
+    } else if (typeof exports !== 'undefined' && typeof module !== 'undefined') {
+        factory(exports, module);
+    } else {
+        var mod = {
+            exports: {}
+        };
+        factory(mod.exports, mod);
+        global.asyncStorage = mod.exports;
```

Comment:
While we're changing so much about how things are defined and imported, it seems like a great opportunity to rename these drivers to something more meaningful (see #270).